packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -356,7 +356,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
       subMenuIconClassName
     );
 
-    if (item.name === '-') {
+    if (item.text === '-' || item.name === '-') {
```

Comment:
nit: wondering if we should perhaps add a comment noting that .name is deprecated and can be removed in the future

Reply:
We should rely on typing for these kinds of changes as this should show up as an error once the field is removed.